os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -175,6 +175,8 @@
  */
 
 static serial_t* sdrv[MAX_UART_INDEX + 1] = {NULL, NULL, NULL, NULL, NULL}; //uart 0~3, uart2 is configured as log uart
+static bool g_tc_rxavailable;
+static bool g_tc_rxavailable_res;	// used to hold the result of rxavailable TC
```

Comment:
I don't agree adding codes here for tc.

Reply:
please check my comment below and let me know what you think?